codecov.yml

```diff
@@ -1,4 +1,10 @@
 ---
+codecov:
+  require_ci_to_pass: true
+  notify:
+    wait_for_ci: true
+    after_n_builds: 9
```

Comment:
Is this needed since you also specify "wait for ci" ?

(Generally, I dislike this option as it creates a risk that we forget to update it, and possibly lose the notification if the workflow is reworked to reduce number of "reports")

Reply:
Yes it is needed, if I only leave `wait_for_ci` it will get a single report, and wait for the job of that particular report to finish, instead of waiting for the whole workflow to finish.

And yes it does create a bit of maintenance, but honestly in my experience number of reports tend to go up, instead of down. So I don't think it's that much of a big deal.